README.md

```diff
@@ -2,34 +2,45 @@
 
 The neovim package specification supports a single, top-level package metadata file. This file can be *either* 'plugin.lua' or 'plugin.json'. The format is loosely based on the [Rockspec Format](https://github.com/luarocks/luarocks/wiki/Rockspec-format) and can contain the following fields:
 
-* `package` : String, the name of the package
+* `package` (String) the name of the package
 
-* `version` : String, the version of the package. Should obey semantic versioning conventions, for example `0.1.0`. Plugins should have a git commit with a `tag` matching this version. For all version identifiers, implementation should check for a `version` prefixed with `v` in the git repository, as this is a common convention.
+* `version` (String) the version of the package. Should obey semantic versioning conventions, for example `0.1.0`. Plugins should have a git commit with a `tag` matching this version. For all version identifiers, implementation should check for a `version` prefixed with `v` in the git repository, as this is a common convention.
 
-* `specification_version` : String, the current specification version. (0.1.0) at this time.
+* `specification_version` (String) the current specification version. (0.1.0) at this time.
 
-* `source` : Table, contains information on how to fetch sources.
-  * `url` (string, mandatory field) : the URL of the package source archive. Examples: "http://github.com/downloads/keplerproject/wsapi/wsapi-1.3.4.tar.gz", "git://github.com/keplerproject/wsapi.git". Different protocols are supported: 
+* `source` (Table|String) contains information on how to fetch sources.
```

Comment:
Nitpick: why not just collapse this to `url (String)`, if that's the only field the table can contain? If we're leaving open the possibility of e.g. `releases_only`, then leaving it like this makes more sense.

Reply:
I put `releases_only` under dependencies, I guess we could collapse here since I don't have a clear usecase for the table anymore